packages/core/src/sdk/session/index.ts

```diff
@@ -50,6 +50,16 @@ export const mutation = gql(`
 `)
 
 export const validateSession = async (session: Session) => {
```

Comment:
At first sight, I found it strange to put the logic here.  
I wonder if the `validateSession` function is the best place to put this logic or change the `storeConfig.session` object before passing it in the initial `createSessionStore(storeConfig.session)` function. But in the `validateSession`, it is more guaranteed since we call it every time, right?

The `validateSession` would be better for cases where the user starts with the `postalCode`, then does the trick to erase it manually, `validateSession` would ensure there is always a default value.

Reply:
Yes! That was the main reason, for the cases in which for whatever reason the postalCode is erased, the default one will still be used